### ISSUE TITLE
[Shader Bug] SEUS shader bug

### ISSUE THREAD

**Issue by NightZore** [sp614x/optifine#7046]
![image_2022-08-02_203743030](https://user-images.githubusercontent.com/110470232/182376392-eaee1729-87a0-4c63-98f3-24f4a75dcaef.png)
Shaders are bugged out when using any of SEUS shaders
no shadows just these bugged lines
Please help fix this issue for 1.19 no 1.19.1 please
Minecraft 1.19
Optifine_1.19_H9
No forge/fabric

**Comment by mindforger** [comment]
looks like this nevidia driver bug, 516 version does have this artifact if i remember right

**Comment by NightZore** [comment]
> looks like this nvidia driver bug, 516 version does have this artifact if i remember right

So its an Nvidia Driver issue not a bug of optifine?

**Comment by fayer3** [comment]
yes that's a driver issue with version `516.40` update to the latest on which should be `516.5`

**Comment by mindforger** [comment]
not sure but check your driver just in case, try updating it or downgrading it
